docs/markdown/setup/ole-db.md

```diff
@@ -0,0 +1,56 @@
+# OLE DB Setup and Support
+
+OLE DB can be used with the Amazon Timestream ODBC driver.
```

Comment:
[The OLE DB to ODBC bridge](https://learn.microsoft.com/en-us/sql/ado/guide/appendixes/microsoft-ole-db-provider-for-odbc?view=sql-server-ver15)](https://learn.microsoft.com/en-us/sql/ado/guide/appendixes/microsoft-ole-db-provider-for-odbc?view=sql-server-ver15) can be used with the Amazon Timestream ODBC driver.

Reply:
Changed.